src/routes/discount.js

```diff
@@ -40,7 +43,9 @@ router.post("/apply", applyDiscountValidationRules(), validate, applyDiscount);
 //create route
 router.post(
   "/",
-  isAuthorized(null, null),
+  isAuthorized(CLASS_ENROLMENT, CREATE, {
```

Comment:
CLASS_ENROLMENT in discount?

Reply:
like i didnt find any relavent pages soo i added this

Reply:
You can create the page if it's not there.

Reply:
ok i have created the page

Reply:
I think you can use `BUSINESS_FINANCE` page in the isAuthorized parameter

Reply:
ok i changed it to business_Finance for now